docs/migrations/v8.md

```diff
@@ -303,6 +303,5 @@ isolation.
 
 #### @zendeskgarden/react-utilities
 
-- This package has been deprecated This package has been removed.and will be removed in a future
-  major release.
+- This package has been deprecated and will be removed in a future major release.
```

Comment:
Not sure how this typo fell in. This is a copy-paste revert from the migration doc on `main`.